public/js/src/interface.js

```diff
@@ -0,0 +1,45 @@
+showHeadlinesAndPictures();
+resetHashUrl();
+showSummaryOnClick();
+
+function showHeadlinesAndPictures() {
+  setTimeout(() => {
```

Comment:
what's the timeout for?

Reply:
I found the API was too slow otherwise and I wouldn't get the articles on the page! If there is a better way to achieve this I would love to know! As I think this may have cause me some bugs, although not sure